src/pages/categories.js

```diff
@@ -1,8 +1,20 @@
 import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { checkStatus } from '../redux/categories/categoriesSlice';
 
-const Categories = () => (
-  <div className="status-div">
-    <button type="submit" className="status-btn">Check Status</button>
-  </div>
-);
+const Categories = () => {
+  const allCategories = useSelector((state) => state.categories);
+  const dispatch = useDispatch();
+
+  const dispatchAction = () => {
+    dispatch(checkStatus());
+  };
+
+  return (
+    <div className="status-div">
+      <button type="submit" className="status-btn" onClick={dispatchAction()}>Check Status</button>
```

Comment:
- [optional] I observed that the **Under construction** text is displayed by default which means the `check status button` is not working as it should, it would be great if you can address this as a learning exercise. 👍

Reply:
Ohhh right! Thanks will make sure it works.😊